### ISSUE TITLE
[java] UnsynchronizedStaticFormatter reports commons lang FastDateFormat

### ISSUE THREAD

**Issue by george-007** [pmd/pmd#1633]
**Affects PMD Version:** 
6.11

**Rule:**
`UnsynchronizedStaticFormatter`

**Description:**
The new rule `UnsynchronizedStaticFormatter` reports  usage of `org.apache.commons.lang3.time.FastDateFormat` which is a commonly used thread safe alternative to `java.text.SimpleDateFormat`. 

Could you add an exception to not check the instances of  `FastDateFormat`?

`UnsynchronizedStaticDateFormatter` will be removed in PMD 7.0. 

**Code Sample demonstrating the issue:**
```
private static final FastDateFormat FDF = FastDateFormat.getInstance("dd.MM.yyyy HH:mm:ss");
```

**Comment by Shubham-2k17** [comment]
I am not able to reproduce the issue. I tried with test case given below & it is working fine. Do inform if i am missing something.
```
<test-code>
        <description>#1633 [java] UnsynchronizedStaticFormatter reports commons lang FastDateFormat</description>
        <expected-problems>0</expected-problems>
        <code><![CDATA[
import org.apache.commons.lang3.time.FastDateFormat;

public class Foo {
    private static final FastDateFormat FDF = FastDateFormat.getInstance("dd.MM.yyyy HH:mm:ss");
}
     ]]></code>
</test-code>
```

**Comment by george-007** [comment]
The violation is reported when the formatter is used.
The complete test case:
```
public class Foo {
    private static final FastDateFormat FDF = FastDateFormat.getInstance();
    public void format() {
        FDF.format(new Date());
    }
}
```

**Comment by Shubham-2k17** [comment]
Hi @jsotuyod,
I created a pull request for this issue. No idea why travis-ci build is failing. I am not getting any error locally.

**Comment by Shubham-2k17** [comment]
found the issue will update.
